cmd/s3-gw/app_settings.go

```diff
@@ -259,7 +260,7 @@ func newSettings() *viper.Viper {
 
 		os.Exit(0)
 	case versionFlag != nil && *versionFlag:
-		fmt.Printf("NeoFS S3 gateway %s\n", version.Version)
+		fmt.Printf("NeoFS S3 Gateway\nVersion: %s\nGoVersion: %s\n", version.Version, runtime.Version())
```

Comment:
Shouldn't we print `Component: NeoFS S3 Gateway` instead of `NeoFS S3 Gateway` as in https://github.com/nspcc-dev/neofs-s3-gw/issues/578#issue-1294811778 ?

Reply:
Couldn't we append `Build: 2022-07-05T21:45:30+00:00`?